React/Fabric/Mounting/ComponentViews/Text/RCTParagraphComponentView.mm

```diff
@@ -8,7 +8,7 @@
 #import "RCTParagraphComponentView.h"
 
 #import <react/components/text/ParagraphComponentDescriptor.h>
-#import <react/components/text/ParagraphLocalData.h>
+#import <react/components/text/ParagraphState.h>
```

Comment:
Those are not alphabetically sorted now.

If you can run clang format on that with config from the repo, that would be great. Otherwise I can do it for you.

Reply:
@shergin ran `yarn clang-format` (complains about a lot of files!) but weirdly, nothing to say about the ordering. manually fixed it.